modules/phase_field/include/kernels/CHBulk.h

```diff
@@ -77,10 +77,8 @@ CHBulk<T>::validParams()
   InputParameters params = KernelGrad::validParams();
   params.addClassDescription("Cahn-Hilliard base Kernel");
   params.addParam<MaterialPropertyName>("mob_name", "M", "The mobility used with the kernel");
-  params.addDeprecatedCoupledVar("args",
-                                 "Vector of variable arguments of the mobility",
-                                 "args is deprecated, use 'coupled_variables' instead");
-  params.addCoupledVar("coupled_variables", "Vector of variable arguments of the mobility");
+  params.addCoupledVar("args", "Vector of variable arguments of the mobility");
+  params.deprecateCoupledVar("args", "coupled_variables", "02/27/2024");
```

Comment:
`coupled_variables` is the new not deprecated parameter.

Reply:
ok, so this is correct

Reply:
Yup - the method arguments are `old_name`, `new_name`, and then `removal_date`.